src/App.js

```diff
@@ -3,20 +3,32 @@ import {
   BrowserRouter as Router,
   Switch,
   Route,
+  Link
 } from 'react-router-dom';
 import BeaniesPage from './BeaniesPage';
 import BeanieDetail from './BeanieDetail';
 
+
 export default function App() {
   return (
     <Router>
       <div className='App'>
+        <ul>
+          <li>
+            <Link to="/">Home</Link>
+          </li>
+          <li>
+            <Link to="beanie-baby">Find</Link>
+          </li>
+        </ul>
         <Switch>
-          <Route>
+          <Route exact path="/">
             {/* this home page route should list all the beanies */}
+            <BeaniesPage />
           </Route>
           <Route> 
```

Comment:
you seem to be missing your `/:id` route here, which I think is why your detail page is not working.